database_create_script

```diff
@@ -49,7 +49,7 @@ CREATE TABLE IF NOT EXISTS public.payment_methods
     id bigint NOT NULL GENERATED ALWAYS AS IDENTITY ( INCREMENT 1 START 1 ),
     qr_code text,
     upi_id character varying,
-    mobile numeric(10),
+    mobile numeric(10) NOT NULL,
```

Comment:
We might not need payment_methods.phone if qrcode/upi_id there.
Instead we can add unique constraint to all 3 columns.
What do u think @rohit-manethiya